clarifai/models/model_serving/model_config/inference_parameter.py

```diff
@@ -0,0 +1,109 @@
+import json
+from dataclasses import asdict, dataclass, field
+from typing import Any, List
+
+
+@dataclass(frozen=True)
+class InferParamType:
+  BOOL: int = 1
+  STRING: int = 2
+  NUMBER: int = 3
```

Comment:
Very soon we're going to have `ENCRYPTED_STRING = 21;` support which will be needed here. I think it's ok to get ahead and add that now. That would be used for things like API Keys where we never want to return it via the API. The user would provide a string value. I see the code currently depends on their only being one string field type, so maybe we can always default to string, but have a way for users to specify it's an encrypted string too.
cc @wemoveon2

Reply:
yup add `ENCRYPTED_STRING` here and i'll make the changes to BE to support the field type.